src/core/services/theming/theming.js

```diff
@@ -670,9 +670,13 @@ function ThemingProvider($mdColorPalette, $$mdMetaProvider) {
    * @param {object} options Theme definition options
    * Options are:<br/>
    * - `primary` - `{string}`: The name of the primary palette to use in the theme.<br/>
+   * - `primaryHues` - `{object} (Optional)`: Overwrite hues for primary palette.<br/>
    * - `accent` - `{string}`: The name of the accent palette to use in the theme.<br/>
+   * - `accentHues` - `{object} (Optional)`: Overwrite hues for accent palette.<br/>
    * - `warn` - `{string}`: The name of the warn palette to use in the theme.<br/>
+   * - `warnHues` - `{object}`: Overwrite hues for warn palette.<br/>
```

Comment:
Can you please add an example to the main [`$mdTheming` docs](https://github.com/angular/material/blob/v1.1.10/src/core/services/theming/theming.js#L570-L597) where these new parameters are used?

Without that, it's not clear here what this `object` would consist of.